### PR TITLE
Bamboo agent: add additional volumes and mounts

### PR DESCRIPTION
**Pull request by timmy59100**
## Pull request description

Provides support for additional volume and volume mounts for bamboo-agent as described in issue https://github.com/atlassian/data-center-helm-charts/issues/530

## Checklist
- [ ] I have added unit tests
- [ ] I have applied the change to all applicable products
- [ ] The E2E test has passed (use `e2e` label)
- [ ] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

File: src/main/charts/bamboo-agent/templates/deployment-agent.yaml [hunk 1]
```diff
@@ -111,3 +114,5 @@ spec:
       {{- if .Values.schedulerName }}
       schedulerName: {{ .Values.schedulerName  | quote }}
       {{- end }}
+      volumes:
+        {{ include "bamboo-agent.volumes" . | nindent 8 }}
```
Comment (bianchi2) [inline src/main/charts/bamboo-agent/templates/deployment-agent.yaml:118]: Can we rename it to `bamboo-agent.additionalVolumes`?

**Review by bianchi2 — CHANGES_REQUESTED** [review]

**Comment by bianchi2** [comment]
@timmy59100 can you also sign CLA please?

**Comment by timmy59100** [comment]
I put the wrong email in the CLA, i requested a change from the team but didn't hear anything.

**Review by bianchi2 — APPROVED** [review]

**Comment by bianchi2** [comment]
@timmy59100 we're planning a release soon and I'll be happy to include merge your PR so that changes make it to the new release. Were you able to sort out CLA issues?

**Review by nanux — APPROVED** [review]

**Comment by timmy59100** [comment]
@bianchi2 No, I didn't hear from the team. Should I make a new pull request with the correct email?

**Comment by bianchi2** [comment]
@timmy59100 yes, please. Also, please make sure you use the latest main branch in your fork. Thanks

**Pull request closed without merging**